exchanges/ftx/ftx_types.go

```diff
@@ -234,18 +234,18 @@ type WalletCoinsData struct {
 	Name             string `json:"name"`
 }
 
-// BalancesData stores balances data
-type BalancesData struct {
-	Coin  string  `json:"coin"`
-	Free  float64 `json:"free"`
-	Total float64 `json:"total"`
+// WalletBalance stores balances data
+type WalletBalance struct {
+	Coin                   string  `json:"coin"`
+	Free                   float64 `json:"free"`
+	Total                  float64 `json:"total"`
+	AvailableWithoutBorrow float64 `json:"availableWithoutBorrow"`
+	USDValue               float64 `json:"usdValue"`
```

Comment:
Can't seem to find the 'usdValue' field in their api docs anymore. Maybe they mightve removed it